.github/fixtures/test-fixtures-locally.sh

```diff
@@ -32,4 +32,4 @@ fi
 
 # Show results
 echo -e "\n---Run git-cliff---" >&2
-cargo run --manifest-path "$SCRIPT_DIR/../../Cargo.toml" -- -vv --config "$FIXTURES_DIR/cliff.toml" "${@:2}"
+cargo run --manifest-path "$SCRIPT_DIR/../../Cargo.toml" -- -vv --config "$FIXTURES_DIR/cliff.toml" "${@:2}" > "$FIXTURES_DIR/output.md"
```

Comment:
What's the reasoning for this change? Is the file being used anywhere or just for testing purposes?

I'd prefer to see the output in the terminal tbh - maybe `cat` it afterwards?